specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -1202,15 +1499,70 @@
       },
       "description": "Properties of the perimeter associable resources."
     },
-    "LoggingCategory": {
+    "CheckMembers": {
       "type": "object",
       "properties": {
-        "name": {
+        "memberDestinations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MemberDestination"
+          },
+          "x-ms-identifiers": [
+            "privateLinkResourceId"
+          ]
+        }
+      },
+      "description": "Request object for check members API."
+    },
+    "MemberDestination": {
+      "type": "object",
+      "properties": {
+        "privateLinkResourceId": {
+          "type": "string",
+          "description": "Private link resource Id."
+        },
+        "isMember": {
+          "readOnly": true,
+          "type": "string",
+          "description": "yes/no, if it is member or not."
+        },
```

Comment:
use bool or more preferably an enum : 

[https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review)

Reply:
We have string in the backend so we have kept string in swagger also. We will try to change this in the next PR once backend changes are done.

Reply:
That does not matter, backend can have it as string. This will only prevent the clients from passing free form string.

Reply:
Changed this to enum.